### PR TITLE
Remove occurrences of DiffBlue

### PR DESCRIPTION
**Pull request by mgudemann**
- [x] Each commit message has a non-empty body, explaining why the change was made.
- [x] My contribution is formatted in line with CODING_STANDARD.md.
- [x] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: e1962a3a16f165030c3ab2822b4488dd32303564).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85473209

File: jbmc/unit/java_bytecode/ci_lazy_methods/lazy_load_lambdas.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
  Module: Unit tests for parsing generic classes
 
- Author: DiffBlue Limited. All rights reserved.
+ Author: Diffblue Limited. All rights reserved.
```
Comment (peterschrammel) [inline jbmc/unit/java_bytecode/ci_lazy_methods/lazy_load_lambdas.cpp]: Drop `All rights reserved.` (several occurrences).

**Review by kroening — APPROVED** [review]

**Comment by mgudemann** [comment]
@peterschrammel done

**Comment by tautschnig** [comment]
I would appreciate if the "Each commit message has a non-empty body, explaining why the change was made." could be honoured here. Especially here. These are changes where the rationale is impossible to figure out from the behavioural difference, because there is none.

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: b73cf1d3425493f6ba189a76168c88fa2b005b1e).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85527157

**Comment by mgudemann** [comment]
@tautschnig done

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: fa4fd5b517daf01e72962e5c6658eb22a882b503).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85545392

**Review by chrisr-diffblue — APPROVED** [review]

**Pull request merged**